app/controllers/users/registrations_controller.rb

```diff
@@ -1,10 +1,6 @@
 class Users::RegistrationsController < Devise::RegistrationsController
   def create
```

Comment:
Nit: Can this controller action be removed please?

It's only delegating to the superclass version of create action now, and we don't need the override anymore.